content/en/blog/gitops-your-falco-rules/index.md

```diff
@@ -0,0 +1,805 @@
+
+![Successful workflow execution](images/gitops-your-falco-rules-04.png)
+
+Now it's time to test our freshly generated OCI artifact.
+
+## Testing our Rules Workflow
+
+How do we test our new fancy rule distribution system? The best way would be using a customizable Falco installation to pull our rules.
+
+Here we can have two scenarios: running Falco as a Linux service or deploying it as a Kubernetes Daemonset. Let's cover both of them to see the differences.
+
+### Falco and Falcoctl running as Linux services
+
+For this scenario, I'll use the [Try Falco](https://falco.org/try-falco-on-ubuntu/) scenario which instructions are available in the Falco website. Only the first set of instructions are required to have the scenario up and running. Once our scenario is ready, we'll proceed to configure the rules automatic update.
+
+First of all, let's have a look into the `falcoctl-artifact-follow.service` unit status:
+```
+$ sudo systemctl status falcoctl-artifact-follow.service
+
+● falcoctl-artifact-follow.service - Falcoctl Artifact Follow: automatic artifacts update service
+     Loaded: loaded (/lib/systemd/system/falcoctl-artifact-follow.service; disabled; vendor preset: enabled)
+     Active: active (running) since Wed 2023-04-12 15:34:06 UTC; 55s ago
+       Docs: https://falco.org/docs/
+   Main PID: 16152 (falcoctl)
+      Tasks: 8 (limit: 2271)
+     Memory: 10.5M
+     CGroup: /system.slice/falcoctl-artifact-follow.service
+             └─16152 /usr/bin/falcoctl artifact follow --allowed-types=rulesfile
+
+Apr 12 15:34:06 vagrant systemd[1]: Started Falcoctl Artifact Follow: automatic artifacts update service.
+Apr 12 15:34:06 vagrant falcoctl[16152]: INFO: Reading all configured index files from "/root/.config/falcoctl/indexes.yaml"
+Apr 12 15:34:06 vagrant falcoctl[16152]: WARN: No configured index. Consider to configure one using the 'index add' command.
+Apr 12 15:34:06 vagrant falcoctl[16152]: INFO: Creating follower for "falco-rules:0", with check every 6h0m0s
+Apr 12 15:34:07 vagrant falcoctl[16152]:  INFO  Starting follower for "ghcr.io/falcosecurity/rules/falco-rules:0"
+Apr 12 15:34:07 vagrant falcoctl[16152]:  INFO   (ghcr.io/falcosecurity/rules/falco-rules:0) found new version under tag "0"
+Apr 12 15:34:10 vagrant falcoctl[16152]:  INFO   (ghcr.io/falcosecurity/rules/falco-rules:0) artifact with tag "0" correctly installed
+vagrant@vagrant:~$ 
+```
+We can see there that there is an instance of `falcoctl` _following_ an item named **falco-rules:0**. We'll come back to that later.
+
+Let's have a look then into `falcoctl.yaml` configuration file, the default one, since no other explicit configuration file has been passed for the service.
+
+```
+$ cat /etc/falcoctl/falcoctl.yaml 
+
+artifact:
+  follow:
+    every: 6h0m0s
+    falcoVersions: http://localhost:8765/versions
+    refs:
+    - falco-rules:0
+indexes:
+- name: falcosecurity
+  url: https://falcosecurity.github.io/falcoctl/index.yaml
```

Comment:
We need to explain how to use a protected registry, how to set up the credentials, it's important to address secured envs